packages/web-api/src/methods.ts

```diff
@@ -221,11 +222,13 @@ export abstract class Methods extends EventEmitter<WebClientEvent> {
   public abstract apiCall(method: string, options?: WebAPICallOptions): Promise<WebAPICallResult>;
 
   public readonly admin = {
+    // TODO: admin.analytics.getFile
```

Comment:
As described at https://github.com/slackapi/node-slack-sdk/issues/1191 , this endpoint returns binary data. We will work on it separately.